OracleBI/dockerfiles/12.2.1.3/Dockerfile

```diff
@@ -50,7 +50,7 @@ COPY $BI_DISTRO_ZIP1 $BI_DISTRO_ZIP2 install.file oraInst.loc /u01/
 USER root
 
 # tools for installing and running BI
-RUN yum -y install unzip hostname make && yum clean all
+RUN yum -y install unzip hostname && rm -rf /var/cache/yum
```

Comment:
can you move this call to the other RUN statement?

Reply:
Also, make sure you clean up yum cache after, to keep the image small. See other Dockerfiles for examples

Reply:
This cannot be removed because it has to run as `root`. The other `RUN` statement is run as `oracle`.

Reply:
The `rm -rf /var/cache/yum` command removes the `yum` cache.